es6.md

```diff
@@ -145,3 +155,37 @@ interface ClassExpression <: Class, Expression {
     type: "ClassExpression";
 }
 ```
+
+# Modules
+
+## ExportDeclaration
+
+```js
+interface ExportDeclaration <: Declaration {
+    default: boolean;
+    declaration: VariableStatement | Declaration | HoistableDeclaration | null;
```

Comment:
@mikesherov Looks like it's from the ES6 grammar, should be `VariableDeclaration` though obviously.